contracts/acl/ACL.sol

```diff
@@ -136,6 +136,32 @@ contract ACL is IACL, AragonApp, ACLHelpers {
         _setPermissionManager(_newManager, _app, _role);
     }
 
+    /**
+     * @notice Get parameters for permission array length
+     * @param _entity Address of the whitelisted entity that will be able to perform the role
+     * @param _app Address of the app
+     * @param _role Identifier for a group of actions in app
+     * @return Length of the array
+     */
+    function getPermissionParamsLength(address _entity, address _app, bytes32 _role) external view returns (uint) {
+        return permissionParams[permissions[permissionHash(_entity, _app, _role)]].length;
+    }
+
+    /**
+    * @notice Get parameters for permission
+    * @param _entity Address of the whitelisted entity that will be able to perform the role
+    * @param _app Address of the app
+    * @param _role Identifier for a group of actions in app
+    * @param _index Index of parameter in the array
+    * @return Parameter (id, op, value)
+    */
+    function getPermissionParams(address _entity, address _app, bytes32 _role, uint _index) external view returns (uint8 id, uint8 op, uint240 value) {
```

Comment:
Since we can only get a single param at a time, might make more sense to drop the plurals (e.g. `getPermissionParam`).

Reply:
Makes sense.